src/App.tsx

```diff
@@ -1,4 +1,6 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
```

Comment:
For the future, better not to use such short names, you can name it `classnames` here

Reply:
well, I saw that we name it 'cn' in Misha's examples + It takes less space, and in my opinion, it's pretty understandable what it means

Reply:
@polinavafik @Moroz-Dmytro I think you can use both options, on real projects usually such things are simply discussed in the team and adhere to the same style